src/components/UIShell/GlobalHeader/HeaderAction.svelte

```diff
@@ -2,10 +2,11 @@
   export let type = undefined;
   export let icon = undefined;
   export let isOpen = undefined;
+  export let text = undefined;
 
   let elRigthPanel = undefined;
 
-  import { cx } from '../../../lib';
+  import { cx, optional } from '../../../lib';
```

Comment:
Remove the "optional" import

Reply:
I must have forgotten to remove that. Done!